<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZFlyEmSplitUploadOptionDialog</class>
 <widget class="QDialog" name="ZFlyEmSplitUploadOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Option</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="autoAnnotationCheckBox">
     <property name="statusTip">
      <string>Pass current annotation to split parts</string>
     </property>
     <property name="text">
      <string>Pass Annotation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="statusCheckBox">
       <property name="statusTip">
        <string>Check the box to set new status</string>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="commentCheckBox">
       <property name="statusTip">
        <string>Check the box to add new comment</string>
       </property>
       <property name="text">
        <string>Comment</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commentLineEdit">
       <property name="statusTip">
        <string>Comment to be added to the annotation of the split parts</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hintLabel">
     <property name="statusTip">
      <string>Hint: Annotation will be passed to the split parts if 'Pass Annotation' is on, but it's comment/status field will be replaced if 'Comment'/'Status' is on.</string>
     </property>
     <property name="text">
      <string>&lt;font color=&quot;DarkGreen&quot;&gt;Hint&lt;/font&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZFlyEmSplitUploadOptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZFlyEmSplitUploadOptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
